lib/Entity/Manager.php

```diff
@@ -274,6 +275,14 @@ public function fieldKeys()
                 $tableKeys['index'][$fieldKeyName][] = $fieldName;
                 $usedKeyNames[] = $fieldKeyName;
             }
+            if ($fieldInfo['fulltext']) {
```

Comment:
I need some tests for this - a new entity that defines a fulltext column that is also an index.